test/mocks/grpc/mocks.h

```diff
@@ -91,5 +102,36 @@ class MockAsyncClientManager : public AsyncClientManager {
                                      Stats::Scope& scope, bool skip_cluster_check));
 };
 
+MATCHER_P(ProtoBufferEq, expected, "") {
```

Comment:
Check out https://github.com/envoyproxy/envoy/blob/4b0b6540a1c34423813c6a81967cd6656c7b8f05/test/test_common/utility.h#L542.

Reply:
Checked the result of Parse.  PTAL

Reply:
Maybe rename to `ProtoInBuffer` or something, I keep reading this as `ProtobBufEq`, trying to avoid poison tipped needles.